seismic/inventory/add_time_corrections.py

```diff
@@ -0,0 +1,154 @@
+#! /usr/bin/env python
+"""
+Add GPS clock time correction csv_data into inventory file to get a modified station xml file
+
+CreationDate:
+    24/02/2020
+
+Developer:
+    fei.zhang@ga.gov.au
+
+"""
+
+import os
+import sys
+from obspy import read_inventory
+from obspy import Inventory
+from obspy.core.inventory import Network
+from obspy.core.util import AttribDict
+
+
+def get_csv_correction_data(path_csvfile):
+    """
+    Read in the csv data from an input file, get the network_code, station_code, csv_data
+
+    Args:
+        path_csvfile: input csv file in /g/data/ha3/Passive/SHARED_DATA/GPS_Clock/corrections/
+        $ head 7D.DE43_clock_correction.csv
+        net,sta,date,clock_correction
+        7D,DE43,2012-11-27,1.0398489013215846
+        7D,DE43,2012-11-28,0.9408504322549281
+        7D,DE43,2012-11-29,0.8418519631882714
+        7D,DE43,2012-11-30,0.7428534941216148
+        7D,DE43,2012-12-01,0.6438550250549583
+
+    Returns: (network_code, station_code, csv_data)
+
+    """
+    with open(path_csvfile, "r") as csvfid:
+        all_lines = csvfid.readlines()
+
+    print("The csv file length", len(all_lines))
+    print("the first line", all_lines[0])
+    print("the last  line", all_lines[-1])
```

Comment:
Print statements can be removed, or tidied up and replaced with logging which offers more control.